GTG/gtk/preferences.py

```diff
@@ -165,35 +156,22 @@ def valid_check(self, widget):
         self.refresh_hour.modify_fg(Gtk.StateFlags.NORMAL, color)
         self.refresh_mins.modify_fg(Gtk.StateFlags.NORMAL, color)
 
-    def on_close(self, widget, data=None):
-        """ Close the preferences dialog."""
-        self.valid_check(widget)
-        self.timer.time_changed()
-        self.dialog.hide()
-        return True
-
-    @classmethod
-    def on_help(cls, widget):
-        """ Open help for preferences """
-        help.show_help("preferences")
-        return True
-
     @classmethod
-    def on_autostart_toggled(cls, widget):
+    def on_autostart_toggled(self, widget, state):
```

Comment:
If you keep method marked as `@classmethod`, the first argument should be `cls`, not `self` (You might even drop the decorator if you want to)